wallet/db_queries.c

```diff
@@ -0,0 +1,13 @@
+#include "config.h"
+#include "db_postgres_sqlgen.c"
+
+#if HAVE_POSTGRES
```

Comment:
Seems this code is duplicated with the one in `wallet/db_queries_postgres.c`.

Reply:
Hmm yeah, I think I can clean this up even more. Let me give it a try...